neo/io/axonio.py

```diff
@@ -296,7 +296,7 @@ def read_block(self, lazy=False, cascade=True):
 
                     anaSig = AnalogSignal(signal, sampling_rate=sampling_rate,
                                           t_start=t_start,
-                                          name=name.decode('utf-8'),
+                                          name=str(name),

```

Comment:
ok, changing to unicode for all IOs is a big effort, so I think that should be done in a separate PR. If you're otherwise ok with these fixes, can you merge this?